pages/studentAssessmentInstanceExam/studentAssessmentInstanceExam.ejs

```diff
@@ -115,129 +110,47 @@
-                <%- include('../partials/scorebar', {score: assessment_instance.score_perc}) %>
-              </div>
-              <div class="col-md-6 col-sm-12 col-xs-12">
-                Time completed: <%= assessment_instance.closed_at %>
-              </div>
-            </div> <!-- row -->
-          </div> <!-- well -->
-
         </div> <!-- panel-body -->
 
         <table class="table table-condensed table-hover">
           <thead>
             <tr>
               <th>Question</th>
-              <th class="cell-center">Number of attempts</th>
+              <th class="cell-center">Best submission</th>
+              <th class="cell-center">Available points</th>
               <th class="cell-center">Awarded points</th>
             </tr>
           </thead>
           <tbody>
-            <% questions.forEach(function(question) { %>
-            <% if (question.start_new_zone && question.zone_title) { %>
+            <% instance_questions.forEach(function(instance_question) { %>
+            <% if (instance_question.start_new_zone && instance_question.zone_title) { %>
             <tr>
-              <th colspan="3"><%= question.zone_title %></th>
+              <th colspan="5"><%= instance_question.zone_title %></th>
             </tr>
             <% } %>
             <tr>
               <td>
-                <a href="<%= urlPrefix %>/instance_question/<%= question.id %>/">
-                  <%= question.question_number %>. <%= question.question_title %>
+                <a href="<%= urlPrefix %>/instance_question/<%= instance_question.id %>/">
+                  Question <%= instance_question.question_number %>
                 </a>
               </td>
-              <td class="cell-center"><%= question.number_attempts %></td>
-              <td class="cell-center"><%= question.points %>/<%= question.max_points %></td>
+              <td class="cell-center"><%- include('../partials/examQuestionScore', {instance_question}); %></td>
+              <td class="cell-center"><%- include('../partials/examQuestionAvailablePoints', {open: assessment_instance.open && instance_question.open, points_list: instance_question.points_list}); %></td>
+              <td class="cell-center"><%- include('../partials/examQuestionAwardedPoints', {points: instance_question.points, max_points: instance_question.max_points}); %></td>
             </tr>
             <% }); %>
           </tbody>
         </table>
 
-        <!-- ######################################################################################### -->
-        <!-- ######################################################################################### -->
-        <!-- ######################################################################################### -->
-        <!-- ######################################################################################### -->
-        <% } %> <!-- assessment_instance.open -->
+        <div class="panel-footer">
```

Comment:
Consider putting this information in popovers?

Reply:
Done.